Japan: fix + cleanup

KingBlueSapphire opened the pull request:
**This update does not modify gameplay**
- updated to protocol 1.5.0
- fixed bug where players could not destroy blocks that were over void (leaves and logs)
- moved enchantments to inline
- removed yaw definition in default region, because default value is 0
- updated yaw values to be in spawn element
- sorted min and max values for readability
- replaced filters with inline filters
Credit to @PeakDominance for assistance with the XML!

KingBlueSapphire committed:
Japan: fix + cleanup

- updated to protocol 1.5.0
- fixed bug where players could not destroy blocks that were over void (leaves and logs)
- moved enchantments to inline
- removed yaw definition in default region, because default value is 0
- updated yaw values to be in spawn element
- sorted min and max values for readability
- replaced filters with inline filters

TheRealPear commented on ctw/japan/map.xml:
```diff
@@ -35,62 +33,46 @@
+    <apply enter="green" use="green" region="red-for-green" message="You may not enter your team's own wool room!"/>
+    <apply enter="orange" use="orange" region="cyan-for-orange" message="You may not enter your team's own wool room!"/>
+    <apply block="deny(only-beacon)"/>
+    <apply block="deny(void)" region="void-area" message="You may not place or break blocks here!"/>
```
This should be moved so that it is the last region application, per https://pgm.dev/docs/guides/xml-pointers/filter-apply-order

calcastor commented on ctw/japan/map.xml:
```diff
@@ -35,62 +33,46 @@
         </regions>
     </default>
-    <spawn team="green" kit="spawn-kit">
+    <spawn team="green" kit="spawn-kit" yaw="-90">
```
You should change here `<regions/>` in this to `<region/>`; `<regions/>` is for collections of point regions that can have individual yaws.

I would prefer that you keep the yaw within the `<region/>` as defined here, because yaws can be per spawn region.

So, for this:


```xml
    <spawn team="green" kit="spawn-kit">
        <region yaw="-90">
            <point>62.5,11,21.5</point>
        </region>
    </spawn>
```

calcastor commented on ctw/japan/map.xml:
```diff
@@ -35,62 +33,46 @@
+    <apply enter="green" region="green-spawn" message="You may not enter the enemy's spawn!"/>
+    <apply enter="green" use="green" region="red-for-green" message="You may not enter your team's own wool room!"/>
+    <apply enter="orange" use="orange" region="cyan-for-orange" message="You may not enter your team's own wool room!"/>
+    <apply block="deny(only-beacon)"/>
```
I think you mean `use`?

calcastor commented on ctw/japan/map.xml:
```diff
@@ -35,62 +33,46 @@
-        <material>beacon</material>
-    </deny>
+    <material id="only-chest">chest</material>
+    <material id="only-beacon">beacon</material>
```
More of a suggestion, these could just be named `chest` and `beacon` but I could see why you may not want to do so.

limefox3054 replied:
This suggestion was implemented in #116!

KingBlueSapphire committed:
Japan: fix corrections

- moved enchants out of inline
- updated filter apply order
- changed yaw to be in region element
- corrected beacon to use

KingBlueSapphire commented:
Also fixed observers spawn position so players don't get stuck in the Lilypad on newer versions.
